packages/core/integration-tests/test/globals.js

```diff
@@ -0,0 +1,18 @@
+import assert from 'assert';
+import path from 'path';
+import {bundle, outputFS} from '@parcel/test-utils';
+
+describe('global alias', function() {
+  it('should support global alias syntax', async function() {
+    let b = await bundle(
+      path.join(__dirname, '/integration/global-alias/index.js'),
+    );
+
+    let index = await outputFS.readFile(
+      b.getBundles().find(b => b.name.startsWith('index')).filePath,
+      'utf8',
+    );
+
+    assert(/module\.exports\ =\ React/.test(index));
```

Comment:
Testing this using  `module.exports = React.createElement...` and
```js
let output = await run(b, {React: {createElement(tag){ return "ok"; }}});

assert.equals(output, "ok");
```
would be less brittle.

Reply:
Fixed in 825a3eace291139a66f74e3df8d5a072488fbb97.